homework/05/instagram_api/src/main/java/com/example/demo/Login.java

```diff
@@ -0,0 +1,47 @@
+package com.example.demo;
+
+import jakarta.servlet.annotation.WebServlet;
+import jakarta.servlet.http.HttpServlet;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+@WebServlet(name = "login", value = "/login")
+public class Login extends HttpServlet {
+    private String username;
+    private int password;
+
+    private static UserInfo logged_user;
+
+    private boolean exists = false;

```

Comment:
`username`, `password` and `exists` are not connected to the class and are only used in the `doPost` method. You could move them inside the method.